src/features/loupe/BoundingBoxLabel.jsx

```diff
@@ -131,7 +132,7 @@ const BoundingBoxLabel = forwardRef(({
       catSelectorOpen={catSelectorOpen}
       selected={selected}
       css={{
-        backgroundColor: labelColor.base,
+        backgroundColor: displayLabel?.color,
```

Comment:
I'm seeing that you fixed the drawing object issue, but the color of the box changes after you release the mouse. Can you make the temporary bounding box the same color that's used for the box while the user's drawing it?